config/locales/en.yml

```
@@ -1244,7 +1244,7 @@ en:
       disabled_account: Your current account will not be fully usable afterwards. However, you will have access to data export as well as re-activation.
       followers: This action will move all followers from the current account to the new account
       only_redirect_html: Alternatively, you can <a href="%{path}">only put up a redirect on your profile</a>.
-      other_data: No other data will be moved automatically
+      other_data: No other data will be moved automatically (includes your posts and the list of accounts you follow)
```

```suggestion
      other_data: No other data (including your posts and the list of accounts you follow) will be moved automatically
```

This improves clarity and readability to me?

---

Maybe?

Here's an alternative:
```suggestion
      other_data: No other data (e.g., your posts and the list of accounts you follow) will be moved automatically
```

I'm slightly worried that the `No` and `including` will not be properly parsed by readers. I can't quite explain it, but that's why I pushed it to the end.

---

Haha. I think the pushing to the end created that confusion for me. Language is hard :)

---

Anyway, think over the two choices w/ the parenthetical in the middle. I'll apply whichever you pick.